practitioner-service/src/main/java/uk/ac/ox/ndph/arts/practitioner_service/service/PersonService.java

```diff
@@ -0,0 +1,48 @@
+package uk.ac.ox.ndph.arts.practitioner_service.service;
+
+import java.util.UUID;
+
+import org.hl7.fhir.r4.model.IdType;
+import org.hl7.fhir.r4.model.Practitioner;
+import org.hl7.fhir.r4.model.Enumerations.AdministrativeGender;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.ac.ox.ndph.arts.practitioner_service.model.Person;
+import uk.ac.ox.ndph.arts.practitioner_service.repository.IFhirRepository;
+import uk.ac.ox.ndph.arts.practitioner_service.exception.HttpStatusException;
+import uk.ac.ox.ndph.arts.practitioner_service.exception.ArgumentException;
+
+/**
+* Implement an IEntityService.
+* validation is for empty values on all fields.
+*/
+@Service
+public class PersonService implements IEntityService {
```

Comment:
Any special reason this is Person and not Practitioner?

Reply:
wasn't sure if this is a practitioner for the rest of the system, or is it a person. the user story wasn't clear on it. do you think it's model should be practitioner?